src/components/ContentWidgets/ToDoList/ToDoListEditor.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { WidgetEditorProps } from "../../ContentMapping/ContentMapping";
+import { ToDoList } from "./ToDoList";
+
+/**
+ * To do list editor allows making changes to a list of checked or unchecked items
+ *
+ * Last Modified
+ * Jennifer Tao
+ * July 6, 2019
+ */
+export const ToDoListEditor: React.FC<WidgetEditorProps> = ({
+  editedContent,
+  setEditedContentOnChange
+}) => {
+  let toDoList_content = editedContent.toDoList_content || [{}];
+
+  let handleText = (e: any, index: number) => {
+    const newToDoList = toDoList_content.slice();
+    newToDoList[index].text = e.target.value;
+    setEditedContentOnChange("toDoList_content", newToDoList);
+  };
+
+  let handleCheck = (index: number) => {
+    const newToDoList = toDoList_content.slice();
+    newToDoList[index].isChecked = !toDoList_content[index].isChecked;
+    setEditedContentOnChange("toDoList_content", newToDoList);
+  };
+
+  let handleAdd = () => {
```

Comment:
I would probably rename these functions to something more human understandable like `addToDoItem` instead of `handleAdd` or `checkedToDoToggle` instead of `handleCheck`